makesteel.lic

```diff
@@ -22,10 +22,11 @@ class MakeSteel
     args = parse_args(arg_definitions)
     total_count = args.count.to_i
     type = args.type || 'h'
-    @hometown = get_settings.hometown
+    @settings = get_settings
```

Comment:
This does not need to be an instance var (@) since it is not used outside this function

Reply:
Yes, @settings is not used outside this function. I think I changed it after I made @hometown. For consistency, should I remove @hometown = and set all @hometown references to @settings.hometown?

Reply:
I think that's ok here